src/main/java/server/request/RequestParser.java

```diff
@@ -0,0 +1,33 @@
+package server.request;
+
+public class RequestParser {
+  private String request;
+
+  public Request parse(String request) {
```

Comment:
`request` in method signature looks confusing, it receives request and return request but in different form. Can we rename it to `rawRequest` or`httpMessage`?

Reply:
Yes! Good point, will do now